content/blogposts/2023/announcing-scip-clang.md

```diff
@@ -0,0 +1,84 @@
+# Announcing scip-clang: a new SCIP indexer for C and C++
+
+[scip-clang](https://github.com/sourcegraph/scip-clang) is our new indexer
+for C and C++ code written from the ground up to natively emit [SCIP](https://github.com/sourcegraph/scip)
+and especially support the wide range of language features present in C++.
+
+scip-clang is now available in beta, and replaces our older indexer lsif-clang.
+We've successfully indexed large codebases such as Chromium with scip-clang.
+
+Here's a quick demo showcasing some features:
+
+{/* TODO: Add demo here */}
+
+scip-clang supports a superset of the functionality of lsif-clang.
+The main additions are:
+
+- scip-clang uses a multi-processing architecture instead of multi-threading,
+  so that indexing failures (such as crashes) in a single translation unit
+  do not affect indexing for other translation units.
+- scip-clang natively supports code navigation for `#include` pragmas and macros.
+- scip-clang is based on Clang 16 instead of Clang 11.
+  It consumes Clang as a library rather than as a fork,
+  making it easy to update the version of Clang used in the future. 
+
+Some other minor quality-of-life improvements include:
+
+- scip-clang infers paths to standard headers for GCC and Clang
+  from the compilation database without requiring extra command-line flags.
+- scip-clang binaries are available for both Linux and macOS.
+- Despite supporting more features, thanks to SCIP, index sizes
+  (both compressed and uncompressed) are about
+  10%-20% the size of the corresponding LSIF indexes.
+  This translates to faster uploads, a lower likelihood of upload errors,
+  and reduced risk of out-of-memory when the Sourcegraph backend
+  processes an index.
+- scip-clang uses incremental parsing for compilation databases,
+  reducing the risk of out-of-memory errors on ingestion.
+
+If you are a Sourcegraph customer using C or C++,
+please [try out scip-clang](https://github.com/sourcegraph/scip-clang),
```

Comment:
Perhaps just say "please try...".  Prospects might read this too etc?

Reply:
Reworded.